### PR TITLE
[FIX] TIC in getting started

### LINKED ISSUE #111: getting started confuses total ion current and total ion chromatogra,
needs rework

### PR DESCRIPTION
**Pull request by oliveralka**
fixes #111

### CODE REVIEW

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -218,16 +218,16 @@ slower):
...
 and MS2 spectra separately (generally only MS1 TIC is provided).
 
-With this information, we can write a function that calculates the total ion
-current (TIC) for a given ms level: 
+With this information, we can write a function that calculates the TIC for a given ms level: 
 
 .. code-block:: python
 
-    # Calculates total ion current of an LC-MS/MS experiment
+    # Calculates total ion chromatogram of an LC-MS/MS experiment
```
Comment (timosachsenberg) [inline docs/source/getting_started.rst:230]: this calculates the current, right? not the chromatogram

Comment (oliveralka) [reply docs/source/getting_started.rst:230]: Yes, you are right - I misunderstood. 

A TIC (Total Ion Chromatogram) is a chromatogram created by summing up intensities of all mass spectral peaks belonging to the same scan. So basically at a specific retention time. 

The total ion current (TIC) chromatogram represents the summed intensity across the entire range of masses being detected at every point in the analysis.

**Comment by timosachsenberg** [comment]
Explanation still seems a bit off, mainly because in literature both terms and abbreviations are often mixed.
Would it make sense to have:
1. calculate total ion current of the whole experiment
2. calculate the total ion current chromatogram (=TIC)

**Comment by oliveralka** [comment]
Ok, as far as I have seen you are doing both already. 

1. calculate total ion current of the whole experiment
```
 >>> calcTIC(exp, 1)
    240.0
```
2. calculate the total ion current chromatogram (=TIC)
```
    tic = exp.calculateTIC()
    retention_times, intensities = tic.get_peaks()
# plot retention times and intensities and add labels
    plt.plot(retention_times, intensities)
```
I can rephrase it further - to help with the distinction.

**Review by timosachsenberg — APPROVED** [review]

**Pull request merged**